docs/advanced/naming.md

```diff
@@ -1,160 +1,198 @@
-- Including the Episode title is highly recommended in this case as it makes identification significantly easier and understandable for anyone who downloads it. Neither `S00EXX` or `0.5` provides this information at a glance which is why the episode title is really helpful in this case. Refer to either [thetvdb](https://thetvdb.com/) or [MAL](https://myanimelist.net/) to find the episode name.
-- You may include the absolute number of the episode as long as it's not a decimal episode that lands between any two existing ones. Optional and really just down to preference. It's much better than decimal episode numbering but still not ideal.
-- This means we need a name that is easily readable, parsable, and identifiable. This is achieved by using a combo of the above.
+![Releases using bad naming schemes](/static/advanced/naming/releases-bad-naming.png)
 
-```
-Anime Name - S00E04 - Title of the Episode
-```
+These naming schemes are **bad** as they:
 
-- This is an example of a well named special episode. `Anime Name` tells us the name of the anime, `S00E04` tells us the episode number in the `SXXEXX` standard which makes it easily identifiable, and `Title of the Episode` makes it significantly more human readable.
+Do not include the [type of release](/guides/quality/#types-of-releases) and [where it is sourced from](/guides/quality/#blu-ray-vs-web)
+:   - File 2 does not list the video codec/resolution, audio codec, "Blu-ray/WEB" source, dual audio, etc.
 
-```
-Anime Name - 18.5 - S00E05 - Title of the Episode
-```
+May be too long in length
+:   - File 2 includes episode chapters in the file name, which results in a long file name
 
-- Another good example, this tells us the name of the anime, where it lands chronologically, what episode it is with reference to databases and the name of the episode to aid with identifying the episode without much effort.
+Use an invalid naming scheme
+:   - File 1 separates the season and episode number (`S2 - 08`). *As a result, some programs may incorrectly parse this as `S01E08`*
+    - File 3 uses brackets to hold the season and episode number (`[S02,E00]`). *As a result, some programs may incorrectly parse this as a release group*
 
-#### More Examples
+Alter or uses a condensed show title
+:   - File 3 uses the incorrect order for the show title (`From Me to You Kimi Ni Todoke` instead of `Kimi ni Todoke - From Me to You`)
+    - File 4 uses a condensed show title (`Masamune` instead of `Masamune-kun's Revenge` or `Masamune-kun no Revenge`)
 
-Bad, do not use:
+Using good naming schemes, it could look like the following:
 
-```
-[Group] Anime Name - S02OVA02.mkv
-[Group] Anime Name - 18.5 [BD 1080p FLAC] [CRC].mkv
-[Group] Anime Name - 06.5 (SP) [BD 1080p FLAC].mkv
-[Group] Anime Name - Title of the Episode [CRC].mkv
-[Group] Anime Name - 17.5 (OVA) [CRC].mkv
-```
+![Releases using good naming schemes](/static/advanced/naming/releases-good-naming.png)
```

Comment:
Too many examples cause confusion. reduce it down to just the two variants that are recommended ahead.
1. `[smol] Yuri Kuma Arashi - S01E01 (BD 1080p HEVC Opus) [Dual Audio] [CF1029D9].mkv`
2. `Whisper.Me.a.Love.Song.S01E06.1080p.WEBRip.DDP2.0.x265-smol.mkv`